internal/server/server_test.go

```diff
@@ -6,10 +6,11 @@ import (
 	"encoding/json"
 	"fmt"
 	"github.com/google/uuid"
+	"github.com/quan-to/chevron/internal/agent"
 	"github.com/quan-to/chevron/internal/config"
-	"github.com/quan-to/chevron/internal/etc"
 	"github.com/quan-to/chevron/internal/etc/magicbuilder"
 	"github.com/quan-to/chevron/internal/keymagic"
+	"github.com/quan-to/chevron/internal/tools"
 	"github.com/quan-to/chevron/pkg/QuantoError"
 	"github.com/quan-to/chevron/pkg/interfaces"
 	"github.com/quan-to/chevron/test"
```

Comment:
Same comment for [imports](https://github.com/golang/go/wiki/CodeReviewComments#imports)